modules/integration/tests-integration/src/test/java/org/wso2/mdm/integration/device/enrollment/WindowsEnrollment.java

```diff
@@ -104,24 +105,9 @@ public void testWSETP() throws Exception {
         Assert.assertEquals(response.getResponseCode(), HttpStatus.SC_ACCEPTED);
     }
 
-
     private String readXML(String fileName, String characterEncoding) throws Exception {
         URL url = ClassLoader.getSystemResource(fileName);
         File folder = new File(url.toURI());
```

Comment:
Is the variable name sound like it's referring to a folder? As I feel it's pointing to a XML file and if so, variable name should be refactored meaningfully.